packages/playground/_stories/main/Label/Label.stories.ts

```diff
@@ -29,45 +29,85 @@ const Template: UI5StoryArgs<Label, StoryArgsSlots> = (args) => {
 	?required="${ifDefined(args.required)}"
 	?show-colon="${ifDefined(args.showColon)}"
 	wrapping-type="${ifDefined(args.wrappingType)}"
+	class="${ifDefined(args.className)}"
 >
 	${unsafeHTML(args.default)}
 </ui5-label>`;
```

Comment:
please remove the newlines here:
```ts
>${unsafeHTML(args.default)}</ui5-label>`;
```
it causes some extra whitespace that #7042 describes

Reply:
All on one line